settings.gradle

```diff
@@ -0,0 +1,11 @@
+import static java.lang.Boolean.parseBoolean
+
+rootProject.name = 'toolbox'
+
+// Composite build setup for compiling hoist-core locally
+if (parseBoolean(runHoistInline)) {
+    println "${xhAppName}: running with Hoist Core INLINE...."
+    includeBuild '../hoist-core'
```

Comment:
If we are including static pathing, we should 100% ensure the documentation denotes where things should live. **_It may_**, I just want to confirm we are thinking about this.